ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnerGroups.vue

```diff
@@ -134,6 +135,14 @@ export default defineComponent({
           : []
       }),
       groupFractionalData: find(getTransferOrRegistrationHomeOwnerGroups(), { groupId: props.groupId }),
+      fractionalOwnershipData: computed((): FractionalOwnershipWithGroupIdIF => {
+        // for New Group (the last one in dropdown), reset interest values
+        if (localState.groupItems.length === localState.ownerGroupId) {
+          props.fractionalData.interestNumerator = null // eslint-disable-line vue/no-mutating-props
+          props.fractionalData.interestDenominator = null // eslint-disable-line vue/no-mutating-props
+        }
+        return props.fractionalData
+      }),
```

Comment:
There was a previous explicit requirement to put in default interest values for New Groups, so i'm confused by this a little. 

I guess the question is: 
Has this requirement changed/since been reversed? Can we confirm this from all parties? ie design/qa etc

If so, we should just remove the code that applies the defaults, which i think is code block starting line 603 in add/edit owner component

Reply:
This was raised by Divya. I was also surprised by this, so just waiting for clarification from her.